Intersect (Core)/GameObjects/Maps/MapBase.cs

```diff
@@ -45,30 +52,19 @@ public MapBase(Guid id) : base(id)
             Name = "New Map";
 
             //Create empty tile array and then compress it down
-            if (Layers == null || Layers[0].Tiles == null)
+            if (Layers == null)
             {
-                Layers = new TileArray[Options.LayerCount];
-                for (var i = 0; i < Options.LayerCount; i++)
-                {
-                    Layers[i].Tiles = new Tile[Options.MapWidth, Options.MapHeight];
-                    for (var x = 0; x < Options.MapWidth; x++)
-                    {
-                        for (var y = 0; y < Options.MapHeight; y++)
-                        {
-                            Layers[i].Tiles[x, y] = new Tile();
-                        }
-                    }
-                }
-
-                TileData = mCeras.Compress(Layers);
+                Layers = new Dictionary<string,Tile[,]>();
+                //TODO: Add new layers from config? Remove layers if they no longer exist? Idk!
```

Comment:
What is this todo for exactly?
Isn't this exactly what the proposed changes do?

Reply:
I actually handled that logic elsewhere.. I just pushed a commit to remove the comment.